src/index.ts

```diff
@@ -5,19 +5,26 @@ import URL from 'url';
 import path from 'path';
 import fs from 'fs';
 import { promisify } from 'util';
+import csv from 'csvtojson';
 
 const app = express();
 const port = 3000;
 
 const fsReadFile = promisify(fs.readFile);
 
 const config = {
-  source: "./test-data.csv",
-  mapping: [{
-    fieldName: '',
-    jsonPath: []
-  }]
-};
+  source: __dirname + "/test-data.csv",
```

Comment:
We should provide a way of loading this config from a file. The path could be a CLI arg or a `process.env` value. For the latter would recommend using `dotenv` package